pkg/minishift/provisioner/minishift_provisioner_test.go

```diff
@@ -70,3 +71,69 @@ func TestCentOSImage(t *testing.T) {
 		t.Fatal("Provisioner should detect CentOS")
 	}
 }
+
+func TestMinishiftProvisionerGenerateDockerOptionsForRHEL(t *testing.T) {
```

Comment:
Looking good, the only suggestion I would have is that these two test methods seems identical apart of the release info. So one could extract everything below generation of the info into a new method which could be called by each test to avoid code duplication.